src/init.c

```diff
@@ -406,23 +410,31 @@ static void *init_stdio_handle(const char *stdio, uv_os_fd_t fd, int readable)
         break;
     case UV_NAMED_PIPE:
         handle = malloc(sizeof(uv_pipe_t));
+        JL_UV_LOCK();
         if ((err = uv_pipe_init(jl_io_loop, (uv_pipe_t*)handle, 0))) {
+            // JL_UV_UNLOCK() equivalent is done during unwinding
```

Comment:
Errr, no. This is not the case for the NOGC version of the lock.

Reply:
I think the locks here can just be removed; threads haven't been started yet at this point.